moveit_experimental/moveit_jog_arm/src/low_pass_filter.cpp

```diff
@@ -46,17 +46,19 @@ static const std::string LOGNAME = "low_pass_filter";
 namespace moveit_jog_arm
 {
 LowPassFilter::LowPassFilter(double low_pass_filter_coeff)
+  : filter_coeff_(low_pass_filter_coeff)
+  , scale_term_(1. / (1. + low_pass_filter_coeff))
+  , feedback_term_(-low_pass_filter_coeff + 1.)
```

Comment:
Consider changing the order here to make that minus harder to miss. 
Like this: `feedback_term_(1. - low_pass_filter_coeff)`.